safe_transaction_service/history/serializers.py

```diff
@@ -321,6 +334,16 @@ def validate(self, data):
 class SafeDelegateSerializer(SafeDelegateDeleteSerializer):
     label = serializers.CharField(max_length=50)
 
+    def get_valid_delegators(self, ethereum_client: EthereumClient, safe_address: ChecksumAddress,
+                             delegate: ChecksumAddress) -> List[ChecksumAddress]:
+        """
+        :param ethereum_client:
+        :param safe_address:
+        :param delegate:
+        :return: Valid delegators for a Safe. A delegate shouldn't be able to add itself
+        """
+        return self.get_safe_owners(ethereum_client, safe_address)
```

Comment:
This is a bit confusion as this appears to be returning owners when the method should return delegators 🤔

Reply:
Owners are the delegators (they delegate in a `delegate`)